### DISCUSSION TITLE
Ideas for Station Webpage, QSO Requesting, New Importer

### DISCUSSION THREAD

**Discussion by NoobKeksTV** [wavelog/wavelog#901] [Ideas]
Just found some time to write this topic for the ideas alredy brought up in another post. Here is what i maybe have thought of:

_**A Station Website (like a qrz.com Page)**_
You probably dont want to replace qrz.com and thats not the idea but what if you could add something like a description, maybe pictures, etc. to the public page we alredy have about our stations that we can share.
Like, we alredy have a QSO Map, QSL List, OQRS, etc. Maybe a little description page would be nice to? 

It could be embedded in say Home-Pages, Shared with others easier, etc. and directly feature all other WaveLog specific features (oqrs, the map, etc.) on there directly

_**A QSO-Request Feature / Embed?**_
I saw this with some other Stations a few times now. They always refer to some system where you can enter your call onto their webpage and it enters you into a list so the operator knows that you are there and they call you instead of you going under in 20 people answering a cq call of a station.

It might not be super-useful for all users but may be a nice addition to the oqrs and help managing a small pile up :)

(Edit: FOUND AN ADIF EXPORT OPTION.... I WAS BLIND AF)
_**An .edi File Importer**_
Comming from the Contest side of things i mainly used the Software "Win-Contest" to log all my Contest QSO. This software only provides me with an .edi file on export which i sadly cant directly import into WL. I found a python library that, with some help, converts it into an .adif file which i then can import into WL but its a bit of a Tedeous process. :/ 
I dont know if many software dont support .adif export but i think it would be a nice addition to WL in general

**Comment by AndreasK79** [comment]
@NoobKeksTV could you give me a link to Win-Contest? I've been trying to find it, but no success so far.

**Reply by HB9HIL** [reply]
@AndreasK79 I think I found it https://dd3ku.privatedns.org/

**Reply by NoobKeksTV** [reply]
oh, yes thats it.
i think its mainly in german, i dont know if it supports other languages (since i never used others :p) 
i can provide some sample logs & the convert tool i used if you want to look into it.

**Reply by AndreasK79** [reply]
According to info on the page, it says adif export...

**Reply by NoobKeksTV** [reply]
> According to info on the page, it says adif export...

i...
i went and looked all over the software again cus i never found it. I also looked at the website and indeed it says that.... But i couldnt find it.  Then i messaged the author and... He showed me where.
I am sorry, i was extremly blind here... 🙈🙊

**Reply by AndreasK79** [reply]
No problem. It's not always easy to know or find all functions for a program. At least exporting and importing will be much easier now :)

**Comment by NoobKeksTV** [comment]
What i also just thought of being nice:

On the Public page side of things, a way to share a certain qso with someone. Like "hey look at this qso i did" and u can share an url that leads to maybe a "dedicated" page, showing info about the qso and a map (locator to locator) ? or similar to the internal logbook opening the qso list and then having a modal showing the info with map, etc.

**Reply by NoobKeksTV** [reply]
Similar to this, What about a feature to share say all QSO for a specific period / contest.
Like "hey, heres my contest log" and then u only share all qso from that contest with a link rather than all ur qso u ever did.
Or does this alredy exist?

**Comment by HB9RYZ** [comment]
Hello All
Here some ideas:
1. Ask Synolgy how to create a "Wavelog" Package where the Synology Onwer can download and install Wavelog as a ready package
2. Some links for the section "Tools" in Wavelog

https://sat.fg8oj.com/
https://dxheat.com/dxc/
https://dr2w.de/dx-propagation/
https://www.satmatch.com/
https://sdr-list.xyz/
https://ft8.live/
https://hamalert.org/
https://www.dailydx.com/

73 de HB9RYZ
Wolfgang

**Reply by NoobKeksTV** [reply]
hmmm i dont know my way arround synology but isnt it "just" using docker packages or do they have their own thingy?
on that note, wouldnt it be easier to flash ur synology completely and run a "normal" linux on it or something like truenas or unraid/proxmox which then can run docker container, the file sharing, etc.?

**Reply by HB9RYZ** [reply]
No way, I will not destroy my business Synology RS815+ NAS with another Linux, just to have Wavelog running on my NAS. I have a small spare 2-Bay Synology NAS which I could use - but I have no idea how to install another Linux on a Synolgy NAS.

**Reply by NoobKeksTV** [reply]
> No way, I will not destroy my business Synology RS815+ NAS with another Linux, just to have Wavelog running on my NAS. I have a small spare 2-Bay Synology NAS which I could use - but I have no idea how to install another Linux on a Synolgy NAS.

yeah, thats reasonable 😅 hm, for me i got it running on a small dedicated machine i have., its running a lot of software for me alredy so i just threw it on there. Its a rented machine so its alredy in the public and well, ig this works fine as is.

Regarding the tools: 
Something like a greyline map or a propagation estimate thingy would be very nice too. been using some that google gave me but the results have been .... not thaaat good so far. maybe thats also a result of my rather "bad" setup but i cant say it doesnt work at all, had some amazing qso alredy.
